slicer/SemiologyVisualization.py

```diff
@@ -156,19 +162,25 @@ def getInclusionsWidget(self):
     self.seizureSemiologyCheckBox.setEnabled(False)  # not implemented yet
 
     self.epilepsyTopologyCheckBox.setToolTip(
-      'When the paper selects a sample of patients based on their established'
-      ' epileptogenic zone and/or site of surgical resection (seizure onset zone),'
-      ' and describes the related seizure semiology, e.g. papers looking at TLE, FLE, OLE, etc.'
+      'When the paper pre-selects samples of patients based on their established epileptogenic zone'
+      '(site of surgical resection) or seizure onset zone (neurophysiological/anatomical), and'
+      'describes the related seizure semiology - e.g. articles looking at TLE, FLE, OLE.'
+      ''
```

Comment:
This line doesn't do anything. You can use `\n`, but using two paragraphs in a tooltip seems excessive.

Reply:
I think it's important for the user as some exclude all the cases, others only if it is the only critera.

Reply:
I just meant that line 168 doesn't change the tooltip, I didn't mean that the text was useless. Maybe it wasn't clear, I don't know why 4 lines show up, sorry!